_scripts/redis.js

```diff
@@ -1,3 +1,6 @@
 const RedisServer = require('redis-server');
 const server = new RedisServer(process.env.port);
 server.open().then(() => {});
+process.on('exit', (code) => {
+  server.kill();
```

Comment:
@deeptibaghel do we need to worry that here it is `.close` and we use kill https://www.npmjs.com/package/redis-server#promise-style-close ?